app/helpers/application_helper.rb

```diff
@@ -288,30 +296,28 @@ def flot_bar_chart(name, xaxis_label, yaxis_label, data, options = {})
                 }.merge(options))
   end
 
-  def action_buttons(*args)
-    toolbar_action_buttons args
-  end
-
   def select_action_button(title, options = {}, *args)
     # the no-buttons code is needed for users with less permissions
     return unless args
```

Comment:
While touching this, would it makes sense to reduce the duplicities for args checking here and in `action_buttons`?